### ISSUE TITLE
YARL does not support link-local ipv6 addresses

### ISSUE THREAD

**Issue by socketpair** [aio-libs/yarl#143]
Like this one:

`http://[fe80::822a:a8ff:fe49:470c%enp4s1]:1234/keys`

**Comment by asvetlov** [comment]
'fe80::822a:a8ff:fe49:470c%enp4s1' is not recognized by `ipaddress` standard library.
From my perspective it should be fixed by upstream, otherwise we need add and support parsers for IPv4 and IPv6 addresses. I'd like to avoid it.

**Comment by asvetlov** [comment]
PR is welcome :)

**Comment by socketpair** [comment]
In Python 3.6:
```
In[29]: urlsplit('http://[fe80::822a:a8ff:fe49:470c%qwe]:1234/keys')
Out[29]: SplitResult(scheme='http', netloc='[fe80::822a:a8ff:fe49:470c%qwe]:1234', path='/keys', query='', fragment='')
```
So, it understands URL correctly.

**Comment by asvetlov** [comment]
Fixed by #144
